Tests/SonarScanner.MSBuild.Tasks.UnitTest/WriteProjectInfoFileTests.cs

```diff
@@ -450,7 +450,7 @@ public void GetProjectGuid_WhenProjectGuidEmptyStringAndSolutionConfigurationCon
 
         private void AssertProjectGuidIsRandomlyGenerated(string projectGuid, string solutionConfigurationContents, string fullProjectPath)
         {
-            
+
             // Arrange
```

Comment:
NITPICK: also in this method there are two empty lines before //Act. I guess one can be removed.

Reply:
I'll fix this in the next cleanup PR